### PR TITLE
RBTA-24 login error design/trusevich

### PR DESCRIPTION
**Pull request by itrusevich**
https://jira.pasv.us/browse/RBTA-24
Added tests for login->error->design section.

### CODE REVIEW

File: test/specs/reg/login/error.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { assert } from 'chai';
+import sel from '../../../selectors/login';
+import exp from '../../../expected/login';
+
+
+describe('Design', function () {
+
+    it('Background Color', function () {
+        browser.url('/');
+        $('#login').click();
```
Comment (artsenius) [inline test/specs/reg/login/error.js]: Don't use '#login' here. Put it into selectors and use as variable.

File: test/specs/reg/login/error.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        let error = $(sel.error);
+        let bgColor = error.getCSSProperty('background-color').parsed.hex;
+        assert.equal(bgColor, exp.errorBackgroundColor);
+    });
+
+    it('Border Color', function () {
+        browser.url('/');
```
Comment (artsenius) [inline test/specs/reg/login/error.js]: Why do you refresh browser and click the button every test instead of verifying everything in one error?

Comment (itrusevich) [reply test/specs/reg/login/error.js]: @artsenius corrected. Please check.

File: test/specs/reg/login/error.js (hunk 1 above)
Comment (itrusevich) [reply test/specs/reg/login/error.js]: Created login selector in selectors file.

**Pull request merged**